editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2258,6 +2259,33 @@ ThemeTypeDialog::ThemeTypeDialog() {
 
 ///////////////////////
 
+// Originally copied and adapted from EditorProperty, try to keep style in sync.
```

Comment:
If we have multiple copies of it, should we make it a helper method somewhere? E.g. in EditorHelp as static? CC @YuriSizov

Reply:
That's a good point. It can be a method of `EditorHelpBit`, since this is what we use for these tooltips. I would probably go one step further still. It seems we generate cache for descriptions in every individual control that needs to display those tooltips. It feels excessive to me, and I think we could create `EditorHelpTooltip` that would have a static internal cache which is reused by all these elements.

Reply:
I agree, but I think this should be its own PR.

Reply:
Could we first make that other PR though, instead of duplicating a bunch of code to remove it later?